modules/ROOT/pages/constraints/syntax.adoc

```diff
@@ -193,6 +204,17 @@ Where `<TYPE>` is one of the following property types:
 * `ZONED DATETIME`
 * `DURATION`
 * `POINT`
+* `LIST<BOOLEAN NOT NULL>`
```

Comment:
As per the previous comments, we should indicate what is 5.9 and 5.10.

Reply:
Do we think just having the ` and extended in 5.10` is enough or do we need to be clearer on what it is extended with?

Reply:
It would be nice to have a pill or something to flag the 5.10 ones. e.g. 

![image](https://github.com/neo4j/docs-cypher/assets/1756429/63707fd2-4acd-46bb-9753-d015713b0580)

Not sure if the docs team have support for something like that?

Reply:
I tried, haven't built the docs and the auto build is still broken so I don't know if it worked 🤷

Reply:
Seems to work:

![image](https://github.com/neo4j/docs-cypher/assets/1756429/dfcf3299-fb43-463a-b598-cc34c4f85888)